src/cs/LionWeb.Integration.WebSocket.Tests/WebSocketClientTests.cs

```diff
@@ -83,10 +83,10 @@ [new CommandSource("participationId", "commandId")],
     public async Task Communication()
     {
         var clientA = new WebSocketClient("A");
-        clientA.Receive += (sender, msg) => Console.WriteLine($"client A received: {msg}");
+        clientA.ReceiveFromRepository += (sender, msg) => Console.WriteLine($"client A received: {msg}");
```

Comment:
The `+=` operation “stacks” `EventHandler`s auto-magically?

Reply:
Yes, that's how C# events work.